datafusion/sql/tests/sql_integration.rs

```diff
@@ -233,6 +233,25 @@ fn plan_create_table_with_pk() {
     let sql = "create table person (id int, name string, primary key(id))";
     let plan = r#"
 CreateMemoryTable: Bare { table: "person" } constraints=[PrimaryKey([0])]
+  EmptyRelation
+    "#
+    .trim();
+    quick_test(sql, plan);
+
+    let sql = "create table person (id int primary key, name string)";
+    let plan = r#"
+CreateMemoryTable: Bare { table: "person" } constraints=[PrimaryKey([0])]
+  EmptyRelation
+    "#
+    .trim();
+    quick_test(sql, plan);
+}
+
+#[test]
+fn plan_create_table_with_multi_pk() {
+    let sql = "create table person (id int, name string primary key, primary key(id))";
```

Comment:
I am not sure this means a multi part primary key -- postgres errors on this syntax

```sql
postgres=# create table person (id int, name varchar primary key, primary key(id));
ERROR:  multiple primary keys for table "person" are not allowed
LINE 1: ...e table person (id int, name varchar primary key, primary ke...
```

I think the way this is supposed to be expressed is like

```sql
create table person (id int, name varchar,  primary key(name,  id));
```

Reply:
> I am not sure this means a multi part primary key -- postgres errors on this syntax
> 
> ```sql
> postgres=# create table person (id int, name varchar primary key, primary key(id));
> ERROR:  multiple primary keys for table "person" are not allowed
> LINE 1: ...e table person (id int, name varchar primary key, primary ke...
> ```
> 
> I think the way this is supposed to be expressed is like
> 
> ```sql
> create table person (id int, name varchar,  primary key(name,  id));
> ```

I would expect that `name` is primary key, and `id` is primary key (they each consists of unique values, unlike their tuple version is unique which is also the case.) for the query
```sql
create table person (id int, name string primary key, primary key(id))
```
 In short, when one want to use composite primary key, they have to use 
```sql
create table person (id int, name varchar,  primary key(name,  id));
```
syntax. However, for defining single primary keys, they have alternative version. However, if this behaviour is misleading, or counter intuitive. We can follow postgres and give an error. However, It seems to me that, there is no reason to restrain.

Reply:
@parkma99  is this comment still outstanding?

Reply:
Sorry, I am at home this week.

Reply:
no worries and no pressure! I was just trying to understand what the state of this PR was -- it looks like you are aware of the feedback and just haven't had a chance to implement it. I wanted to make sure you weren't waiting on additional feedback

I'll mark it as draft so it doesn't appear on the list of PRs needing review

Reply:
I still think two primary keys on the same column doesn't really make sense, but we can always make this check more strict as a follow on PR I think